packages/console/src/containers/AppContent/components/Topbar/DocumentNavButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import DocumentIcon from '@/assets/images/document-nav-button.svg';
+import Button from '@/components/Button';
+import DangerousRaw from '@/components/DangerousRaw';
+import useDocumentationUrl from '@/hooks/use-documentation-url';
+
+import * as styles from './index.module.scss';
+
+function DocumentNavButton() {
+  const { documentationSiteUrl } = useDocumentationUrl();
+  return (
+    <Button
+      type="text"
+      size="small"
+      className={styles.documentNavButton}
+      icon={<DocumentIcon className={styles.icon} />}
+      title={<DangerousRaw>Docs</DangerousRaw>}
+      onClick={() => {
+        window.open(documentationSiteUrl, '_blank');
```

Comment:
Using a button element here, user won't be able to open the link in a new window using the right-click button. 
<img width="284" alt="image" src="https://github.com/logto-io/logto/assets/36393111/a1e2d5df-6049-414f-a164-648089981fd5">